app/src/marketplace/containers/ProjectEditing/ProjectEditor.tsx

```diff
@@ -53,7 +53,7 @@ export const ProjectEditor: FunctionComponent<ProjectEditorProps> = ({
 }) => {
     const { state: project } = useContext(ProjectStateContext)
     const { publishInProgress } = useContext(ProjectControllerContext)
-    const { chainId } = getCoreConfig().projectRegistry
+    const chainId = defaultChainConfig.id
```

Comment:
I might leave a config value like `projectsChain` here so we can easily swap main project chain if needed.

Reply:
Seems to me that this would fall a bit into the YAGNI rule. Let's separate it when it will be needed